config/database.yml

```diff
@@ -24,10 +24,11 @@ default: &default
 development:
   <<: *default
   database: killy_recipe
-  # username: postgres
+  username: postgres
   # password: 'killy2468'
-  # host: localhost
-  # port: 5432
+  password: 'psql888tm'
```

Comment:
Kindly remove any comments from your code as this will make your code look neater